deployment/ansible/group_vars/all

```diff
@@ -14,3 +16,7 @@ postgresql_package_version: "9.4.1*.pgdg14.04+1"
 postgresql_host: "localhost"
 postgresql_port: 5432
 postgresql_version: "9.4"
+
+db_uri: "'sqlite:///' + os.path.join(BASE_DIR, 'ashlar.db')"
+csrf_session_key: "seekrit"
```

Comment:
these two should probably only be defined in `development` so that when we forget to override them when we create a production config, we're not pushing insecure keys to production.